libpod/container_attach.go

```diff
@@ -56,16 +56,16 @@ func (c *Container) attachContainerSocket(resize <-chan remotecommand.TerminalSi
 		term.SetRawTerminal(inputStream.Fd())
 	}
 
-	controlPath := filepath.Join(c.bundlePath(), "ctl")
-	controlFile, err := os.OpenFile(controlPath, unix.O_WRONLY, 0)
-	if err != nil {
-		return errors.Wrapf(err, "failed to open container ctl file")
-	}
-	defer controlFile.Close()
-
 	kubecontainer.HandleResizing(resize, func(size remotecommand.TerminalSize) {
+		controlPath := filepath.Join(c.bundlePath(), "ctl")
+		controlFile, err := os.OpenFile(controlPath, unix.O_WRONLY, 0)
+		if err != nil {
+			return
+		}
+		defer controlFile.Close()
+
 		logrus.Debugf("Received a resize event: %+v", size)
-		_, err := fmt.Fprintf(controlFile, "%d %d %d\n", 1, size.Height, size.Width)
+		_, err = fmt.Fprintf(controlFile, "%d %d %d\n", 1, size.Height, size.Width)
```

Comment:
Nit: Why not just do 
if _, err := fmt ... ; err !=nil

Reply:
sure, pushed a new version to fix this.